docs/tutorial/security.md

```diff
@@ -82,6 +108,14 @@ improve the security of your application.
 11. [`<webview>`: Verify options and params](#11-verify-webview-options-before-creation)
 12. [Disable or limit navigation](#12-disable-or-limit-navigation)
 13. [Disable or limit creation of new windows](#13-disable-or-limit-creation-of-new-windows)
+14. [Do not use `openExternal` with untrusted content](#14-do-not-use-openexternal-with-untrusted-content)
```

Comment:
The other checklist items have security warnings associated with them in `lib/renderer/security-warnings.js`. If we add a new checklist item here, it would be nice to get it into `security-warnings.js` tool, either in this PR or in a followup